api/build.gradle

```diff
@@ -48,8 +49,18 @@ dependencies {
     testImplementation project(':grpc-testing')
     testImplementation libraries.guava.testlib
 
-    signature libraries.signature.java
-    signature libraries.signature.android
+    signature (libraries.signature.java) {
+        artifact {
+            extension = "signature"
+        }
+    }
+    // Temporarily disabled until StatusException is fixed.
```

Comment:
Should put a TODO here so that people are prompted to fix it instead of interpreting temporarily.